scarlet/source.py

```diff
@@ -207,10 +213,10 @@ def init_extended_source(
             morph, 0, center=center, algorithm="sdss"
         )
 
-    if monotonic:
+    if monotonic is not None:
         # use finite thresh to remove flat bridges
-        prox_monotonic = operator.prox_strict_monotonic(
-            morph.shape, use_nearest=False, center=center, thresh=0.1
+        prox_monotonic = operator.prox_weighted_monotonic(
+            morph.shape, neighbor_weight=monotonic, center=center, min_gradient=0.2
```

Comment:
This doesn't cause the initial footprints to be larger? I think that was the reason that we used strict monotonicity before in the initialization.

Reply:
That's a misnomer: even `operator.prox_strict_monotonic` with `use_nearst=False`, used weighted monotonicity: https://github.com/pmelchior/scarlet/blob/master/scarlet/operator.py#L115